apps/site/src/app/about/getMembers.ts

```diff
@@ -1,28 +1,78 @@
-const FEED_URL = "https://docs.google.com/spreadsheets/d/";
-const SPREADSHEET_KEY = "1DWCOQBlzA3mpa2BXYXPmQrF9_-SpoPFbTdDQuCQ83hU";
-const QUERY = "pub";
-const FORMAT = "output=tsv";
+import { z } from "zod";
+import { cache } from "react";
+import { client } from "@/lib/sanity/sanityClient";
+import { SanityImageReference } from "@/lib/sanity/types";
 
-const dataURL = FEED_URL + SPREADSHEET_KEY + "/" + QUERY + "?" + FORMAT;
+const Person = z.object({
+	_type: z.literal("person"),
+	name: z.string(),
+	profilePic: SanityImageReference.nullable(),
+	socials: z
+		.object({
+			link: z.string(),
+		})
+		.nullable(),
+});
 
-const getSheetsData = async (page: string) => {
-	const response = await fetch(dataURL);
+const Department = z.array(
+	z.object({
+		person: Person,
+		position: z.string(),
```

Comment:
This PR is focused on pulling content from Sanity, so the Zod schema here is just validating what's returned from Sanity.

As far as I can remember, we decided to separate the Person object from organizers to make it more reusable. For example, people may switch to different departments between years, or we may decided to use the Person object as an author field in the future. The profile photo was something we discussed at the time, but ultimately decided that it wasn't worth the extra complexity. Might be something we have to revisit in the future?

cc @samderanova

Reply:
Adding on to what Alex already mentioned, there might be former organizers that we might want to display on other sections of the site in the future (ex. alumni showcases). In situations like those, it would be better to just have one schema for all of our people and then use them in other schemas where needed instead of having two schemas to do the same.